src/controllers/hashtagController.js

```diff
@@ -25,14 +26,28 @@ export async function getHashtagPosts(req, res){
         const result = await hashtagRepository.getHashtagPosts(hashtag);
         const likeUser = await timelineRepository.likesUsersPost();
         const wasShared = await postRepository.shares();
+        const commentsResult = await commentsRepository.getComments(parseInt(userId));
+        commentsResult.rows.forEach(element => {
```

Comment:
preferir fazer o for of pq segundo o samuel, o forEach as vezes pode quebrar o servidor. É raro mas pode quebrar